Add overview docs (#553)
MicrosoftDocs/quantum-docs-pr: articles/overview/why-learn-quantum-computing.md

---
title: Why you should learn quantum computing?
description: 
author: natke
ms.author:  nakersha
ms.date: 10/16/2019
ms.topic: article
uid: microsoft.quantum.overview.why
---

# Why should you learn Quantum Computing?

> Ok, quantum computers are cool. But, why should you learn quantum computing? In this section, we are going to give you some reasons why learning quantum computing today is a great idea.

Reviewer: ```suggestion
Okay, quantum computers are cool. But, why should you learn quantum computing? In this section, we are going to give you some reasons why learning quantum computing today is a great idea.
```

## Get inspired by the quantum world

Even if we are still far from achieving fault-tolerant quantum computers, we are already observing some impactful consequences of quantum computing in classical computer science. In the last few years, many new algorithms were developed based on the techniques used in quantum computing. These are the so-called quantum inspired algorithms and can run on classical computers. Some examples of these algorithms are recommendation systems or genetic algorithms that perform faster than any previous classical method. By learning quantum computing you will also discover a new way of thinking about computation that will shift your perspective on classical programming! 

> Quantum computing is fun!

Reviewer 2: ```suggestion

```

## Quantum computing is fun!

> Even if the preceding reasons don't convince you, you should still consider learning quantum computing, just for fun! Learning how to code quantum programs, and use the properties of quantum mechanics to solve problems, is a challenging but rewarding experience.

Reviewer 2: ```suggestion
In addition to the real-world impact discussed above, we believe quantum computing can be fun. Learning quantum computing is a rewarding experience where you will come away with cutting-edge knowledge and the skills to help build the technology of the future. Check the section: [How can you learn quantum computing?](xref...) to start writing your first quantum programs!  
```
Update to the last version of the Word doc
